parm/atm/obs/config/lgetkf_amsua_n19.yaml

```diff
@@ -29,46 +29,46 @@ obs error:
 obs filters:
 - filter: Bounds Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   minvalue: 100.0
   maxvalue: 500.0
 - filter: Background Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   threshold: 3.0
 - filter: Domain Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   where:
   - variable:
-      name: scan_position@MetaData
+      name: sensorScanPosition@MetaData
```

Comment:
Not required here, but just for @RussTreadon-NOAA 's awareness, this has been deprecated and should now be `MetaData/sensorScanPosition`

Reply:
Same applies to all of the `@` entries, they should be inverted and replaced with `/`

Reply:
Thanks @CoryMartin-NOAA for catching this.  My preference is to use the proper format.  I'll update the `@` entries as you indicate.